src/main.cc

```diff
@@ -18,12 +18,14 @@ int main(int argc, char** argv) {
     //! Mesh handler
     std::unique_ptr<GMSH> mesh(new GMSH());
 
-    //! Read mesh and compute stresses
+    //! Read mesh and compute material points & stresses
     mesh->read_vertices(io->mesh_file_name());
+    mesh->read_elements(io->mesh_file_name());
```

Comment:
Read elements should call `store_elements` at the end of the function, no reason to call another function in main.